huggingface/pytorch/inference/docker/1.12/py3/sdk2.8.0/Dockerfile.neuron

```diff
@@ -0,0 +1,142 @@
+FROM ubuntu:20.04
```

Comment:
on 02/24 the APL team release PyTorch 1.13 support meaning we could use 1.13 instead of 1.12